src/test/java/org/opensearch/knn/index/codec/KNN950Codec/KNN950CodecTests.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index.codec.KNN950Codec;
+
+import org.apache.lucene.codecs.Codec;
+import org.apache.lucene.codecs.perfield.PerFieldKnnVectorsFormat;
+import org.opensearch.index.mapper.MapperService;
+import org.opensearch.knn.index.codec.KNNCodecTestCase;
+
+import java.io.IOException;
+import java.util.Optional;
+import java.util.concurrent.ExecutionException;
+import java.util.function.Function;
+
+import static org.opensearch.knn.index.codec.KNNCodecVersion.V_9_5_0;
+
+public class KNN950CodecTests extends KNNCodecTestCase {
+
+    public void testMultiFieldsKnnIndex() throws Exception {
+        testMultiFieldsKnnIndex(KNN950Codec.builder().delegate(V_9_5_0.getDefaultCodecDelegate()).build());
+    }
+
+    public void testBuildFromModelTemplate() throws InterruptedException, ExecutionException, IOException {
```

Comment:
see if you want to use @SneakyThrows to remove these exceptions from method signature.

Reply:
oh, nice advise, let me replace list of exceptions with a sneakythrows